frontend/src/components/CostTable.vue

```diff
@@ -180,8 +200,8 @@ const props = defineProps({
     default: [],
   },
   chargeType: {
-    type: String as PropType<ChargeType>,
-    default: "",
+    type: Object as PropType<ChargeType[]>,
+    default: [""],
```

Comment:
`default: () => []`
or 
`default: () => [""]`
when a prop is defined as an Object or an Array, its `default` need to be a function to generate the default value.